test/gui/shared/steps/steps.py

```diff
@@ -646,3 +672,24 @@ def step(context):
 def step(context):
     for tabName in context.table:
         test.vp(tabName[0])
+
+
+@Then('the error "|any|" should be displayed')
```

Comment:
We can also mention where this error is to be displayed like you have done in line 466

Reply:
Technically, it looks for an error label that can be found using the selector `{"name": "errorLabel", "type": "QLabel", "visible": 1}`. So, I think it should work for any error in general. 🤔 And IMO mentioning where the error is (for example `... should be displayed in sharing dialog`) brings more confusion.

Reply:
If you think this way is better then that's fine. I just mentioned it because this step seems a bit generic.